openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -361,7 +361,14 @@ def login_user(request):
             _handle_failed_authentication(user, possibly_authenticated_user)
 
         if not is_edly_user_allowed_to_login(request, possibly_authenticated_user):
-            raise AuthFailedError(_('You are not allowed to login on this site.'))
+            # Check if user registered to any sibling sub-organization of this sub-organization
+            # This check is based on :func:`models.EdlyOrganization.enable_all_edly_sub_org_login` flag
+            if user_belongs_to_any_sibling_sub_organization(request, possibly_authenticated_user):
```

Comment:
Follow theses steps in order:

1. Rename method `user_belongs_to_edly_organization` (`openedx/features/edly/utils.py:user_belongs_to_edly_organization`) to `user_belongs_to_edly_sub_organization`
2. Replace all imports and usages of `user_belongs_to_edly_sub_organization` accordingly
3. Create a new similar method `user_belongs_to_edly_organization` which will be used here